tests/test_components/test_custom.py

```diff
@@ -465,7 +465,7 @@ def test_medium_nk(unstructured):
         meds = CustomMedium.from_nk(n=ns, k=-ks, freq=freqs[0])
     med = CustomMedium.from_nk(n=n, k=-k, allow_gain=True)
     meds = CustomMedium.from_nk(n=ns, k=-ks, freq=freqs[0], allow_gain=True)
-    assert med.eps_model(1e14) == meds.eps_model(1e14)
+    assert np.isclose(med.eps_model(1e14), meds.eps_model(1e14))
```

Comment:
Shouldn't we set some tolerance? Are we using single or double for this?

Reply:
Another thing to consider (taken from the documentation):
> The default value is appropriate for expected values of order unity: if the expected values are significantly smaller than one, it can result in false positives. atol should be carefully selected for the use case at hand. A zero value for atol will result in False if either a or b is zero.

Reply:
You could make the eps dynamic, i.e. something like
```python
a = med.eps_model(1e14)
b = meds.eps_model(1e14)
assert np.isclose(a, b, atol=np.spacing(a))
```
Guess that should work for all cases? Might need a factor of two in there, not sure.

Reply:
I will use eps for doubles for the relative tolerance. I didn't think we needed to be very strict here, but why not!

The absolute tolerance is needed only if values are close to 0. The default used in np.isclose is 1e-8, which is sufficient for these values which are random numbers with magnitudes in the range of 1-2.